source/iostreams.tex

```diff
@@ -16114,7 +16114,7 @@
 
 \pnum
 \begin{note}
-\tcode{directory_iterator} is allowed to cache
+\tcode{directory_iterator} can cache
```

Comment:
Please squash this into the earlier commit that is fixed by this change.

Then, force-push.

Reply:
Done.